src/Random-Tests/RandomTest.class.st

```diff
@@ -97,6 +85,205 @@ RandomTest >> testNextInto [
+		equals:
+			#( 528761024 758681390 94083649 1478012613 4088840 1115059306
+			   2042779717 622797868 1500616650 928800955 1343423686 1805140938
+			   1117901253 1513989040 719521570 30676756 1135651211 1374662617
+			   1418604189 1558546178 2036934963 1469788609 1675210031
+			   930793618 188772961 1052037711 171052541 2032278908 202403553
+			   75708106 1377792829 174288089 201854141 1720608057 1967486854
+			   2083579441 490597026 680583225 945560811 507513760 607549553
+			   1797866806 203951532 1089672142 554526441 542925352 1915146450
+			   1775004195 690648339 364834747 1990327879 1794095924 1442837436
+			   1345319037 1596132913 1892584854 560671416 552713900 1293917643
+			   666888990 44807431 1819783738 1839492845 1320968634 1233020759
+			   1652027798 1666652881 1301878961 1180052126 1862889011
+			   103301227 632526271 1366938134 537700714 128954743 1835838196
+			   1508177090 855491486 1079141906 826946563 546549715 1068957646
+			   1107879981 265164428 2041860109 2064340120 1753621213 430710692
+			   678915420 1207248961 611181228 737748018 1662662122 232588994
+			   1680756096 364506899 1071187044 1867280958 757845914 973714776 )
+]
+
+{ #category : #tests }
+RandomTest >> testPrimitiveRandomGeneration4 [
+
+	| state results |
+	results := OrderedCollection new.
+
+	state := DoubleWordArray new: 1.
+	state at: 1 put: 16re8adc5779db42e9b.
+
+	1 to: 100 do: [ :_ |
+	results add: (Random primitiveRandomNumber: state) ].
+
+	self
+		assert: results asArray
+		equals:
+			#( 771710184 1953164226 488841981 289749225 170525166 936237884
+			   2127861015 660022562 411571104 1072430581 1300895138 404120654
+			   1693461639 1161265675 1159916046 1699383526 590791489 1614792561
+			   380847683 946425151 1485116630 592803324 1052103993 270764341
+			   416898789 1734527238 2108511623 2015395670 118054803 1165759679
+			   875171866 186489425 243817470 416226586 1104201647 1901618852
+			   2069864594 623104591 1883005305 1492162720 1621790882 115867491
+			   1153712239 1627932490 1174012976 1933964673 50718916 372282173
+			   1791395680 16917853 1482549295 2055856276 1105419916 688748746
+			   347250337 179760292 746441714 1233355821 1176628523 1777149394
+			   1811595180 1071613084 400747376 1479420494 822510662 1136293981
+			   348589883 981266437 1654934634 1904093407 28358643 1387644566
+			   1514702487 1167643647 1536752962 528921316 1279459800 1135040089
+			   1600783281 1238099943 1474569181 321494157 703455202 86225344
+			   2071264885 1382839784 2109935752 1119490897 1985674430
+			   1699303289 1863896658 874405881 133529688 1659465559 1036352310
+			   1884407954 1466309468 1125486836 1808697514 1618305223 )
+]
+
+{ #category : #tests }
+RandomTest >> testPrimitiveRandomGeneration5 [
+
+	| state results |
+	results := OrderedCollection new.
+
+	state := DoubleWordArray new: 1.
+	state at: 1 put: 16r39352fc390c369b2.
+
+	1 to: 100 do: [ :_ |
+	results add: (Random primitiveRandomNumber: state) ].
```

Comment:
here.

Even all these tests look similar excepting for some values, maybe some common code could be extracted into common methods?